Source/Renderer/CubeMap.js

```diff
@@ -121,25 +120,30 @@ define([
         gl.activeTexture(gl.TEXTURE0);
         gl.bindTexture(textureTarget, texture);
 
-        function createFace(target, sourceFace) {
+        function createFace(target, sourceFace, preMultiplyAlpha, flipY) {
+            // TODO: gl.pixelStorei(gl._UNPACK_ALIGNMENT, 4);
```

Comment:
Did you mean to leave this TODO in?

Reply:
Yes, that was in the original code. I just moved it.